R/utils.R

```diff
@@ -80,25 +80,25 @@
     }
 }
 
-.check_rowTree_present <- function(tree_name, x,
-                                   name = .get_name_in_parent(tree_name) ){
-    if( !.is_non_empty_string(tree_name) ){
+.check_rowTree_present <- function(tree.name, x,
+                                   name = .get_name_in_parent(tree.name) ){
+    if( !.is_non_empty_string(tree.name) ){
         stop("'", name, "' must be a single non-empty character value.",
              call. = FALSE)
     }
-    if( !(tree_name %in% names(x@rowTree)) ){
+    if( !(tree.name %in% names(x@rowTree)) ){
```

Comment:
At the same go, can you fix this line. @ should not be used. Instead, one should use getters of the class. In this case

names(x@rowTree) --> rowTreeNames(x)